lib/virt_autotest/utils.pm

```diff
@@ -231,15 +231,11 @@ sub create_guest {
         my ($autoyastURL, $diskformat, $virtinstall);
         $autoyastURL = $autoyast;
         $diskformat = get_var("VIRT_QEMU_DISK_FORMAT") // "qcow2";
-
-        assert_script_run "qemu-img create -f $diskformat /var/lib/libvirt/images/xen/$name.$diskformat 20G", 180;
-        assert_script_run "sync", 180;
-        script_run "qemu-img info /var/lib/libvirt/images/xen/$name.$diskformat";
-
         $extra_args = "$linuxrc autoyast=$autoyastURL $extra_args";
         $extra_args = trim($extra_args);
+        assert_script_run "mkdir -p /var/lib/libvirt/images/";
```

Comment:
If I remember correctly,  Libvirt creates this  directory "/var/lib/libvirt/images/" by default. You don't need create it.

Reply:
Please also remove the line "assert_script_run "mkdir -p /var/lib/libvirt/images/xen/" in  prepare_guests.pm. It's not needed.

Reply:
No need to run more VR

Reply:
> 

It is best to keep. at lease test whether folder exists.

Reply:
> 
will delete "assert_script_run "mkdir -p /var/lib/libvirt/images/xen/" in prepare_guests.pm", even if it does not effect running.

Reply:
> > 
> 
> It is best to keep "assert_script_run "mkdir -p /var/lib/libvirt/images" in utils.pm. at lease test whether folder exists.

You have specified this path in virt-install command line.  virt-install will test its existence at least. 
Did you manually checked that  "mkdir -p /var/lib/libvirt/images" is necessary  before running virt-install?

We program based on reality. You can keep it if it's necessary in reality.

Reply:
I test with virt-install manually.
it cannot create folder automatically.
So it is necessary to check its existence.

Reply:
I mean that "/var/lib/libvirt/images" does not exist after you install libvirt packages?

Reply:
I mean that "/var/lib/libvirt/images" does not exist after you install libvirt packages?

Reply:
qambj-server04:~ # rpm -qf /var/lib/libvirt/images
libvirt-daemon-7.1.0-6.8.1.x86_64

Reply:
It's supposed to be created by libvirt-daemon package

Reply:
It's supposed to be created by libvirt-daemon package